citi-onboarding/acervusej-onepage
# Feat/header

**Things I've done:**

- Create a fixed responsive header
- Create a responsive hamburger menu

**How to test it:**

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000.
- See if the header works in mobile and desktop view.
- Look at the code to see if it has everything that it should have.
